processor/logdedupprocessor/README.md

```diff
@@ -30,6 +30,7 @@ This processor is used to deduplicate logs by detecting identical logs over a ra
 | interval            | duration | `10s`       | The interval at which logs are aggregated. The counter will reset after each interval.                                                                                                                                                                                                                                                                                                                                                                  |
 | conditions          | []string | `[]`        | A slice of [OTTL] expressions used to evaluate which log records are deduped.  All paths in the [log context] are available to reference. All [converters] are available to use.                                                                                                                                                                                                                                                                        |
 | log_count_attribute | string   | `log_count` | The name of the count attribute of deduplicated logs that will be added to the emitted aggregated log.                                                                                                                                                                                                                                                                                                                                                  |
+| key                | string | `""`        | Field to use for duplication matching.  Support fields from the log `body` or `attributes`.  Nested fields must be `.` delimited. If a field contains a `.` it can be escaped by using a `\`.  See [example config](#example-config-with-deduplication-key).
```

Comment:
I think using an array of fields to include would be a more complete solution that a single value. It's possible someone would want to select a set of keys to determine uniqueness and ignore the remaining values.

We have `exclude_keys` so I think `include_keys` makes sense.

Reply:
I like the idea, the fact that it pairs with exclude_keys is nice.

Reply:
I have implemented include_fields to pair with exclude_fields. Thanks for the suggestion!